registration/src/test/java/org/spine3/samples/lobby/registration/order/Given.java

```diff
@@ -197,36 +206,56 @@ private Command() {
 
         private Event() {}
 
-        /*package*/ static OrderPlaced orderPlaced() {
+        /* package */ static OrderPlaced orderPlaced() {
             return ORDER_PLACED;
         }
 
-        /*package*/ static OrderUpdated orderUpdated() {
+        /* package */ static OrderUpdated orderUpdated() {
             final OrderUpdated.Builder result = OrderUpdated.newBuilder()
                     .setOrderId(ORDER_ID)
                     .addSeat(newSeatQuantity(16))
                     .addSeat(newSeatQuantity(32));
             return result.build();
         }
 
-        /*package*/ static OrderPartiallyReserved orderPartiallyReserved() {
+        /* package */ static OrderPartiallyReserved orderPartiallyReserved() {
             final OrderPartiallyReserved.Builder result = OrderPartiallyReserved.newBuilder()
                     .setOrderId(ORDER_ID)
                     .addSeat(newSeatQuantity(64))
                     .addSeat(newSeatQuantity(128));
             return result.build();
         }
 
-        /*package*/ static OrderReservationCompleted orderReservationCompleted() {
+        /* package */ static OrderReservationCompleted orderReservationCompleted() {
             final OrderReservationCompleted.Builder result = OrderReservationCompleted.newBuilder()
                     .setOrderId(ORDER_ID)
                     .addSeat(newSeatQuantity(256))
                     .addSeat(newSeatQuantity(512));
             return result.build();
         }
 
-        /*package*/ static OrderConfirmed orderConfirmed() {
+        /* package */ static OrderConfirmed orderConfirmed() {
             return ORDER_CONFIRMED;
         }
+
+        /* package */ static OrderTotalsCalculated orderTotalsCalculated() {
+            final OrderTotalsCalculated.Builder builder = OrderTotalsCalculated.newBuilder()
+                    .setOrderId(ORDER_ID)
+                    .setTotal(newMoney(100, USD));
+            return builder.build();
+        }
+
+        /* package */ static OrderExpired orderExpired() {
+            final OrderExpired.Builder builder = OrderExpired.newBuilder()
+                    .setOrderId(ORDER_ID);
+            return builder.build();
+        }
+
+        /* package */ static OrderRegistrantAssigned orderRegistrantAssigned() {
+            final OrderRegistrantAssigned.Builder builder = OrderRegistrantAssigned.newBuilder()
+                    .setOrderId(ORDER_ID)
+                    .setPersonalInfo(newPersonalInfo("Albert", "Einstein", "ein@stein.com"));
```

Comment:
Put a email address with a domain of the establishment where Albert last worked.